dexbot/strategies/base.py

```diff
@@ -593,6 +601,18 @@ def get_lowest_own_sell_order(self, orders=None):
         except IndexError:
             return None
 
+    def get_external_market_center_price(self):
+#      def get_external_market_center_price(self, exchange='gecko', market='BTC/USD',  suppress_errors=False):  
```

Comment:
Remove commented code if not needed

Reply:
can we omit this file for the current pull please. somehow it added. i'm not sure if i should click on the delete button.

Reply:
can we omit this file for merge?